### ISSUE TITLE
Wake from Sleep, Header info gone, games won't launch

### ISSUE THREAD

**Issue by Zodiaksl** [SteamDeckHomebrew/decky-loader#589]
### Please confirm

- [X] I have searched existing issues
- [X] This issue is not a duplicate of an existing one
- [X] I have checked the [common issues section in the readme file](https://github.com/SteamDeckHomebrew/decky-loader#-common-issues)
- [X] I have attached logs to this bug report (failure to include logs will mean your issue may not be responded to).

### Bug Report Description

Hello,

I've been using Decky for a long time now and I've uninstalled it and installed it in the past.

What is happening now, is, Decky is working fine, however, once my Steam Deck goes to sleep, or I put it back to sleep, once it wakes up, the Header bar in SteamOS disappears, all of the icon information along with it such as:

- Search
- Wifi
- Battery
- Time
- Profile icon

When I wake up my Steam Deck, 1 of 2 things will happen

1. I will hear sound being played by the steam deck waking up, but the screen will remain black, even after trying to wake it up multiple times. Having to reboot to solve it.
2. The header bar will disappear with all of the icons. Which would be fine if that's all it was, but then when I try to launch a game, the game will launch and run, but I will still have the black steam loading screen in front of it. Pressing the Steam Button will reveal the game being played, but the Steam loading screen will remain in front. Or the game will load, but the Steam inputs will be unresponsive.

I thought this might have been caused by CSS Lauder, but I've tried disabling / uninstalling everything.

Steam Deck Model: OLED Limited Edition
SteamOS Channel: Beta
SteamOS Version: 3.5.7
OS Build: 20231122.1

Decky Version
Update Channel: Prerelease
Storage Channel: Default
Update Version: 2.11.1

I've tried every combination available
- SteamOS Beta / Decky Prerelease
- SteamOS Beta / Decky Stable
- SteamOS Stable / Decky Prerelease
- SteamOS Stable / Decky Stable

All of them appear to be doing the same thing.

I've also tried
- Disabling all of my plugins
- Uninstalling all of my plugins
- Uninstallng and Reinstalling Decky from Desktop mode

On current SteamOS builds, with JUST SteamOS and Decky installed, this appears to happen.

Thank you very much for any assistance.

### Expected Behaviour

I wish for Decky to work

### SteamOS version

SteamOS Beta 3.5.7 Build 20231122.1

### Selected Update Channel

Prerelease

### Logs

[deckylog.txt](https://github.com/SteamDeckHomebrew/decky-loader/files/14388984/deckylog.txt)

**Comment by TrainDoctor** [comment]
Can you please upload ~/.steam/steam/logs/cef_log.txt and ~/..steam/steam/logs/cef_log.previous.txt ?
Make sure to scrub through them and remove your steam username as it may appear in these logs.
Thanks.

**Comment by ioneng** [comment]
My LCD Deck is acting up the same with latest decky loader.

**Comment by Zodiaksl** [comment]
> Can you please upload ~/.steam/steam/logs/cef_log.txt and ~/..steam/steam/logs/cef_log.previous.txt ? Make sure to scrub through them and remove your steam username as it may appear in these logs. Thanks.

Yes, no problem at all. Here it is.

[cef_log.previous.txt](https://github.com/SteamDeckHomebrew/decky-loader/files/14392073/cef_log.previous.txt)

**Comment by Zodiaksl** [comment]
> Can you please upload ~/.steam/steam/logs/cef_log.txt and ~/..steam/steam/logs/cef_log.previous.txt ? Make sure to scrub through them and remove your steam username as it may appear in these logs. Thanks.

Yes, no problem. Here is the cef_log.txt and cef_log.previous.txt files

I did a text search but did not find any indication of "username" or my actual steam username, odd.

[cef_log.previous.txt](https://github.com/SteamDeckHomebrew/decky-loader/files/14392077/cef_log.previous.txt)
[cef_log.txt](https://github.com/SteamDeckHomebrew/decky-loader/files/14392078/cef_log.txt)

**Comment by Zodiaksl** [comment]
> My LCD Deck is acting up the same with latest decky loader.

Good to hear I'm not the only one, sorry that this is happening to you also. I love Decky :)

**Comment by NeonSurf** [comment]
I have the same problem I tried everything. Did several factory resets figured out it worked fine before installing Decky. Thought it was a plugin so installed onley Decky on a fresh reset and it was the same bug. Uninstalled wiped Decky and it worked as normal again. Was just abut to submit a ticket when i saw this on top. I run SteamOS 3.5.7 Sable with Stable Decky.

**Comment by VodouChai** [comment]
Found a fix, let me know if it works or mor info needed. 
TLDR 
Issue seems to be caused by Animation Changer 
1. Go to steam settings under Customization--Startup movie and uncheck "Use as Wake Movie"
2. Alternatively set the wake movie as the "boot" webm in Animation Changer.
   (this will play the selected boot movie on wake up from sleep or restart)
3. Issue seems to occur only if the "Use as Wake Movie" is set to steam default
   (setting to suspend was hit and miss)
4. DO NOT set the "Use as Wake Movie" to the throbber as this causes multiple issues. 

Longer version.
-Installed Decky Loader and didn't have an issue until I installed EmuDeck, Pause game and Animation changer.
-Games would launch, however the only buttons that would respond where "B" and four system buttons
-Putting the system to sleep played the "suspend" animation set in Animation loader and shows up in steam settings--Customization--Use as wake up Movie. 
-Uninstalled EmuDeck, and restart device, however issue persisted when playing EmuStation or Steam installed games
-Uninstalled Pause game, and restart device, issue persisted, able to swap between games, but the only buttons that would respond where "B" and four system buttons
-Uninstalled Animation changer, restart system, issue stopped. 

-Reinstalled EmuDeck and Pause game, restart system. No issues
-Reinstalled Animation changer, restart game, Issue reoccurred.
-In system settings, noticed the animations from Animation Loader showed up under Customization menu along with boot videos from the steam store (downloaded or not). 
-Changed the "Use as Wake Movie" to the boot from Animation loader, restart the system and header info did not disappear and launched games responded to all buttons.    
|---VERY IMPORTANT---|
-DO NOT SET THE "Use as Wake Movie to the "Suspend" or "throbber" 
-Doing so causes the following additional issues. 
  -Steam will not connect online : this did not fix itself until shutdown. 
  -UI will start to stutter and freeze
  -throbber will take over boot movie and suspend 
  -Decky Loader will lock up until shutdown.

**Comment by ThomasKorpal** [comment]
Happy to see that I'm not alone ! Also have this issue on my OLED Deck

**Comment by bricefriha** [comment]
> Found a fix, let me know if it works or mor info needed. 
> TLDR 
> Issue seems to be caused by Animation Changer 
> 1. Go to steam settings under Customization--Startup movie and uncheck "Use as Wake Movie"
> 2. Alternatively set the wake movie as the "boot" webm in Animation Changer.
>    (this will play the selected boot movie on wake up from sleep or restart)
> 3. Issue seems to occur only if the "Use as Wake Movie" is set to steam default
>    (setting to suspend was hit and miss)
> 4. DO NOT set the "Use as Wake Movie" to the throbber as this causes multiple issues. 
> 
> Longer version.
> -Installed Decky Loader and didn't have an issue until I installed EmuDeck, Pause game and Animation changer.
> -Games would launch, however the only buttons that would respond where "B" and four system buttons
> -Putting the system to sleep played the "suspend" animation set in Animation loader and shows up in steam settings--Customization--Use as wake up Movie. 
> -Uninstalled EmuDeck, and restart device, however issue persisted when playing EmuStation or Steam installed games
> -Uninstalled Pause game, and restart device, issue persisted, able to swap between games, but the only buttons that would respond where "B" and four system buttons
> -Uninstalled Animation changer, restart system, issue stopped. 
> 
> -Reinstalled EmuDeck and Pause game, restart system. No issues
> -Reinstalled Animation changer, restart game, Issue reoccurred.
> -In system settings, noticed the animations from Animation Loader showed up under Customization menu along with boot videos from the steam store (downloaded or not). 
> -Changed the "Use as Wake Movie" to the boot from Animation loader, restart the system and header info did not disappear and launched games responded to all buttons.    
> |---VERY IMPORTANT---|
> -DO NOT SET THE "Use as Wake Movie to the "Suspend" or "throbber" 
> -Doing so causes the following additional issues. 
>   -Steam will not connect online : this did not fix itself until shutdown. 
>   -UI will start to stutter and freeze
>   -throbber will take over boot movie and suspend 
>   -Decky Loader will lock up until shutdown. 

While this is a good workaround, and I didn't know we could do that (I'm gonna do this right now on mine since I find the start up movie to be annoying after the device wakes up), we would still need a proper fix eventually I think
